src/sonic-config-engine/tests/sample_output/buffers-dell6100.json

```diff
@@ -108,134 +108,266 @@
         "Ethernet0|0": {
             "profile" : "[BUFFER_PROFILE|ingress_lossy_profile]"
         },
+        "Ethernet0|3-4": {
+            "NULL" : "NULL"
```

Comment:
need to change all occurrences below to "profile": "NULL"

Reply:
Fixed

Reply:
Fixed